packages/unified-pipeline/src/components/Canvas/Canvas.tsx

```diff
@@ -75,12 +73,12 @@ const CanvasInternal = (props: CanvasProps) => {
     }).then(({ nodes: layoutedNodes, edges: layoutedEdges }) => {
       setNodes([...layoutedNodes, ...childNodes])
       setEdges(layoutedEdges)
-      window.requestAnimationFrame(() =>
+      window.requestAnimationFrame(() => {
         fitView({
```

Comment:
when do we need to call `fitView` here?

Reply:
`fitView` is required to focus the nodes in the centre of the page once initial layout/positions of the node is computed.